### ISSUE TITLE
Digits does not accept integers like 2.x did

### ISSUE THREAD

**Issue by arokettu** [laminas/laminas-validator#401]
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | 3.0.0

#### Summary

Integer values return false in 3.0.0

#### Current behavior

```php
$validator->isValid(1234);         // returns false
```

#### How to reproduce

```php
<?php
$validator = new Laminas\Validator\Digits();

var_dump($validator->isValid(1234));
```

#### Expected behavior

According to the documentation the old behavior is still the correct one:

```php
$validator->isValid(1234);         // returns true
```

#### Fix

either

```php
        $this->setValue((string) $value);
```

should be

```php
        $value = (string) $value;
        $this->setValue($value);
```

or 

```php
if ($value !== $digits) {
```

should be

```php
if ($this->getValue() !== $digits) {
```

**Comment by gsteel** [comment]
Thanks for the report @arokettu - we'll release 3.0.1 addressing this asap 👍

**Comment by gsteel** [comment]
Fixed in #402
